src/GovUk.Education.ExploreEducationStatistics.Data.Model/Services/Interfaces/IFilterItemService.cs

```diff
@@ -6,7 +6,7 @@ namespace GovUk.Education.ExploreEducationStatistics.Data.Model.Services.Interfa
 {
     public interface IFilterItemService : IRepository<FilterItem, Guid>
     {
-        IEnumerable<FilterItem> GetFilterItems(IQueryable<Observation> observations);
+        IEnumerable<FilterItem> GetFilterItems(Guid subjectId, IQueryable<Observation> observations, bool toList);
```

Comment:
This is called `listFilterItems` rather than `toList` in the implementation

Reply:
done